app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/SimpleSearchActivity.java

```diff
@@ -0,0 +1,39 @@
+package edu.grinnell.appdev.grinnelldirectory.Activity;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import edu.grinnell.appdev.grinnelldirectory.R;
+
+/**
+ * SimpleSearchActivity allows the user to search with one text field.
+ */
+
+public class SimpleSearchActivity extends AppCompatActivity {
+    @BindView(R.id.first_name_field) EditText mFirstNameEditText;
+    @BindView(R.id.last_name_field) EditText mLastNameEditText;
+    @BindView(R.id.search) Button mSearchButton;
+
+    @Override protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_simple_search);
+        ButterKnife.bind(this);
+    }
+
+    /**
+     * Search when the search button is pressed
+     *
+     * @param view SimpleSearchActvity's view
+     */
+    @OnClick(R.id.search)
+    void search(View view) {
```

Comment:
@as-f What is it supposed to do when it's clicked? You already have access to the networking calls, so you should use it. Also, please research how you can pass the `List<Person>` to another activity once you get it. Hint: it will have something to do with `Parcelable` or `Serializable` or `Bundle` or something like that.